### DISCUSSION TITLE
Taskbar icons too wide and Explorer check boxes

### DISCUSSION THREAD

**Discussion by Bec-de-Xorbin** [valinet/ExplorerPatcher#976] [Q&A]
After updating to 22000.493.42.26 and trying centered taskbar and then reverting to default settings (At screen edge), taskbar icons got wider. I don't know how to get them back to normal. Also in Explorer every item has a check box now, even when View -> Show -> Item check boxes is not enabled (tried to enable -> disable that, no effect). Downgrading to 22000.493.42.25 didn't work either. Any ideas?

EDIT: I cannot reproduce this under Windows Sandbox so maybe it was user error or something... Anyways, solved it by system restore.

**Comment by TheLBall** [comment]
Having this issue too, it also seems to make the Compact View option not work. Win11 build 22000.527

UPDATE: This doesn't seem to happen on my main PC. Win10 22000.527. The main difference I can think might have an effect is screen resolution.

**Comment by bobarakatx** [comment]
Having the exact same issue

**Comment by valinet** [comment]
Happened to me as well, now that I switched from using the laptop with my 2 4K screens to using the laptop screen alone.

EDIT: I have developed a fix, found where the problem originated from. It will be fixed in the next pre-release, that's for sure.

**Comment by valinet** [comment]
Thanks for bearing with me. I have issued a fix in the latest update.

From CHANGELOG:

Fixed a bug that made certain systems, under certain circumstances, to become stuck in "tablet mode"; symptoms included more spacing between taskbar icons, compact view permanently disabled in File Explorer, and/or item checkboxes permanently enabled in File Explorer (#1022). After installing this update, if you are still stuck with the tablet UI, open the Registry Editor, go to `HKEY_CURRENT_USER\SOFTWARE\Microsoft\TabletTip\ConvertibleSlateModeChanged` and delete the `ConvertibleSlateModeChanged` which most likely is stuck to value 1.

**Reply by VHFG22** [reply]
I couldn't find the `ConvertibleSlateModeChanged` on `HKEY_LOCAL_MACHINE`, I even tried to create a new one to set it to 0 but only errors about not having enough permissions and then I saw the option on the context menu to jump to `HKEY_CURRENT_USER` and there it was, deleted it, no error, restart explorer and it's back to normal. 👌 Thank you for the quick fix.
Just a notice, I don't know if the user is supposed to have permission or not, but if anyone is struggling to find it, just look on HKCU.

**Reply by valinet** [reply]
Yeah, my bad, I should have said HKCU, sorry. I will correct the post. Apologies.
